### PR TITLE
88 containerize app and server

### LINKED ISSUE #88: Containerize app and server
The flutter app and the server should be containerized together with compose.

### PR DESCRIPTION
**Pull request by lukas0820**
closes #88

### CODE REVIEW

File: server/scripts/setup.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 echo "Installing required dependencies..."
```
Comment (n00on) [inline server/scripts/setup.sh:3]: Isn't this a lie now? :D

File: server/scripts/start-server.sh [hunk 2]
```diff
@@ -1,9 +1,3 @@
 #!/bin/bash
 
-# Note this will fail from in a Windows environment due to resolving failures for .exe files. Use WSL context instead
-if [ "$1" == "--sync" ]; then
```
Comment (n00on) [inline server/scripts/start-server.sh:4]: Isn't this still useful for syncing dependencies without doing the setup again?

**Review by n00on — COMMENTED** [review]
I have two comments, but it seems to work well. @lukas0820

File: server/scripts/setup.sh (hunk 1 above)
Comment (lukas0820) [reply server/scripts/setup.sh]: fixed the log now

File: server/scripts/start-server.sh (hunk 2 above)
Comment (lukas0820) [reply server/scripts/start-server.sh]: thats a good point, reverted this change and modiefied Dockerfile instead

**Review by n00on — APPROVED** [review]

**Pull request merged**